documentation/sdk/docs/how-tos/client/get-node-info.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Get Node Information
+description: Sometimes it's needed to get certain info from the Node to determine for example if the node is synced or which features it has enabled. You can get this info from a Client instance.
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- client
+- load balancer
+- node info
+- nodejs
+- python
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from '!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/client/get-info.ts';
+import PythonCode from '!!raw-loader!../../../../../bindings/python/examples/how_tos/client/get_info.py';
+import RustCode from '!!raw-loader!../../../../../sdk/examples/how_tos/client/get_info.rs';
+import AccountClient from '../../_admonitions/_account-client.md'
+
+Sometimes it's needed to get certain info from the Node to determine for example if the node is synced or which features it has enabled. You can get this info from a Client instance.
+
+<AccountClient/>
+
+The following code example will:
+
+1. Create a `Client` which will connect to the [Shimmer Testnet](https://api.testnet.shimmer.network).
+2. Use the created client to get information about the node.
+3. Print the information to the console.
+
+## Code Example
+
+<Tabs groupId="language">
+    <TabItem value="rust" label="Rust">
+        <CodeBlock className="language-rust">
+            {RustCode}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="nodejs" label="Nodejs">
+        <CodeBlock className="language-typescript">
+            {NodejsCode}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="python" label="Python">
+        <CodeBlock className="language-python">
+            {PythonCode}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+## Expected Output
+
+<Tabs groupId="language">
+<TabItem value="rust" label="Rust">
+
+```bash
+InfoResponse {
+    name: "HORNET",
+    version: "2.0.0-rc.6",
+    status: StatusResponse {
+        is_healthy: true,
+        latest_milestone: LatestMilestoneResponse {
+            index: 5792633,
+            timestamp: Some(
+                1687456380,
+            ),
+            milestone_id: Some(
+                "0x5d554e0c20779dae25288efefb33c385b11c2dc6088f9418d3a1fececa1385fc",
+            ),
+        },
+        confirmed_milestone: ConfirmedMilestoneResponse {
+            index: 5792633,
+            timestamp: Some(
+                1687456380,
+            ),
+            milestone_id: Some(
+                "0x5d554e0c20779dae25288efefb33c385b11c2dc6088f9418d3a1fececa1385fc",
+            ),
+        },
+        pruning_index: 4750998,
+    },
+    supported_protocol_versions: [
+        2,
+    ],
+    protocol: ProtocolParametersDto {
+        protocol_version: 2,
+        network_name: "testnet-1",
+        bech32_hrp: Hrp {
+            inner: [
```

Comment:
Any reason you are using Debug instead on Display? This doesn't really make much sense for the user...

Reply:
Mistake from my side. Will fix it 😅

Reply:
Hmm. `InfoResponse` doesn't implement `Display` 🤔